spec/features/add_card_spec.rb

```diff
@@ -0,0 +1,10 @@
+require "rails_helper"
+context "fill fields in form"  do
+    it 'fill fields in form' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.